### PR TITLE
Dockerfile: updated to new parent

### PR DESCRIPTION
**Pull request by bryanlatten**
Aggressive pruning. 

`499.99MB` vs. `627MB`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,24 @@
-FROM behance/docker-nginx:1.2.4
+FROM behance/docker-nginx:3.0
```
Comment (bryanlatten) [inline Dockerfile:1]: new semver from parent

File: README.md [hunk 2]
```diff
@@ -9,37 +9,61 @@ Provides basic building blocks for PHP web applications, available on Docker Hub
...
 `SERVER_MAX_BODY_SIZE` | `SERVER_MAX_BODY_SIZE=4M` | Allows the downstream application to specify a non-default `client_max_body_size` configuration for the `server`-level directive in `/etc/nginx/sites-available/default`
 `REPLACE_NEWRELIC_APP` | `REPLACE_NEWRELIC_APP=prod-server-abc` | Sets application name for newrelic
 `REPLACE_NEWRELIC_LICENSE` | `REPLACE_NEWRELIC_LICENSE=abcdefg` | Sets license for newrelic, when combined with above, will enable newrelic reporting
-`FPM_HEADER_SIZE` | `FPM_HEADER_SIZE=16k` | Sets header size that will be allowed from PHP-FPM
-`FPM_BUFFER_COUNT` | `FPM_BUFFER_COUNT=4` | Sets the number of buffers for PHP-FPM responses
-`FPM_BUFFER_SIZE` | `FPM_BUFFER_SIZE=16k` | Sets the size of buffers for PHP-FPM responses
-`FPM_BUSY_BUFFER` | `FPM_BUSY_BUFFER=16k` | Total size of buffer that can busy sending while not yet fully read. May need to be changed in coordination with buffer size + count
+`PHP_FPM_MEMORY_LIMIT` | `PHP_FPM_MEMORY_LIMIT=256M` | Sets memory limit for FPM instances of PHP
```
Comment (bryanlatten) [inline README.md:66]: @ingluisjimenez @mikeklein13 @kgeorgiev ^^vv

File: container/root/etc/nginx/sites-available/default [hunk 3]
```diff
@@ -31,15 +27,6 @@ server {
     tcp_nopush off;
     keepalive_requests 0;
 
-    # these values must exist for seamless replacement in debug run.d script
-    proxy_buffer_size 128k;
-    proxy_buffers 4 256k;
-    proxy_busy_buffers_size 256k;
```
Comment (bryanlatten) [inline container/root/etc/nginx/sites-available/default:37]: dumped this functionality, never actually worked appropriately enough for FirePHP headers

File: container/root/run.d/03-debug.sh [hunk 4]
```diff
@@ -1,49 +1,9 @@
...
-
-  if [[ $FPM_HEADER_SIZE ]]
-  then
-    FASTCGI_HEADER_SIZE=$FPM_HEADER_SIZE
-  fi
-
-  if [[ $FPM_BUFFER_COUNT ]]
-  then
-    FASTCGI_BUFFER_COUNT=$FPM_BUFFER_COUNT
-  fi
```
Comment (bryanlatten) [inline container/root/run.d/03-debug.sh:21]: goodbye!

**Comment by mikeklein13** [comment]
:+1:

**Comment by kgeorgiev** [comment]
:+1:

File: Dockerfile [hunk 5]
```diff
@@ -36,61 +49,51 @@ RUN apt-get update && \
...
         newrelic-php5 \
-        git \
         && \
     php5dismod xdebug && \
-    php5dismod newrelic
-
-# Build/install any extensions that aren't in trouble-free packaging
-RUN pecl install igbinary-1.2.1 && \
+    php5dismod newrelic && \
+    \
```
Comment (bryanlatten) [inline Dockerfile:55]: packaging this all as a single container layer

**Comment by anshupande** [comment]
:+1:

File: Dockerfile [hunk 6]
```diff
@@ -36,61 +49,53 @@ RUN apt-get update && \
...
     php5dismod xdebug && \
-    php5dismod newrelic
-
-# Build/install any extensions that aren't in trouble-free packaging
-RUN pecl install igbinary-1.2.1 && \
-    echo 'extension=igbinary.so' > /etc/php5/mods-available/igbinary.ini && \
+    php5dismod newrelic && \
+    \
+    # Add Guzzle feature flag to newrelic APM \
+    echo "newrelic.feature_flag = guzzle" >> $CONF_PHPMODS/newrelic.ini \
```
Comment (bryanlatten) [inline Dockerfile]: @ingluisjimenez new feature flag from newrelic ;)

File: Dockerfile [hunk 7]
```diff
@@ -36,61 +49,52 @@ RUN apt-get update && \
...
     php5dismod xdebug && \
-    php5dismod newrelic
-
-# Build/install any extensions that aren't in trouble-free packaging
-RUN pecl install igbinary-1.2.1 && \
-    echo 'extension=igbinary.so' > /etc/php5/mods-available/igbinary.ini && \
+    php5dismod newrelic && \
+    \
+    # Add Guzzle feature flag to newrelic APM \
+    echo "newrelic.feature_flag = guzzle" >> $CONF_PHPMODS/newrelic.ini && \
```
Comment (ingluisjimenez) [inline Dockerfile:57]: :heart_eyes:

File: README.md (hunk 2 above)
Comment (ingluisjimenez) [inline README.md:66]: good to note that this would be capped to whatever is set on the Container on Mesos

**Comment by ingluisjimenez** [comment]
:+1:

**Pull request merged**